app/src/controllers/appInfo/sagas.js

```diff
@@ -43,6 +45,13 @@ function* updateSessionExpiration({ payload: value }) {
   );
 }
 
+function* updateFooterLinks({ payload }) {
+  yield put(
+    updateServerSettingsAction({ key: SERVER_FOOTER_LINKS_KEY, value: JSON.stringify(payload) }),
+  );
+  yield put(fetchAppInfoAction());
```

Comment:
Please update this flow to local redux update or synchronous call of `fetchAppInfoAction`.
Also apply changes for `updateSessionExpiration` saga.